techniques/system/server-roles/1.0/password-check.cf

```diff
@@ -219,7 +219,7 @@ bundle agent root_password_check_dav
 
   classes:
 
-      "dav_cant_connect" not => returnszero("${g.rudder_curl} --tlsv1.2 --proxy '' ${g.rudder_verify_certs_option} --silent --fail --output /dev/null --user ${g.davuser}:${p.dav_password[2]} --upload-file /opt/rudder/etc/uuid.hive https://localhost/inventory-updates/uuid.hive","noshell");
+      "dav_cant_connect" not => returnszero("${g.rudder_curl} --tlsv1.2 --proxy '' ${g.rudder_verify_certs_option} --silent --fail --output /dev/null --user ${g.davuser}:${g.davpw} --upload-file /opt/rudder/etc/uuid.hive https://localhost/inventory-updates/uuid.hive","noshell");
```

Comment:
this won't work immediately on root, we can maybe do a special case for it.

Reply:
this work for root, same password as generated id ok, we only use value in rudder-passwords.conf in this file, to generate rudder properties

Reply:
Juste we don't set the new password now before it can be propagated to all nodes (needs a new generation to do so), but it's better to have hte same password everywhere